### PR TITLE
Convert Latte to twig

### PR DESCRIPTION
**Pull request by TomasVotruba**
With help of https://github.com/symplify/lattetotwigconverter

### CODE REVIEW

File: source/skoleni/phpstorm.twig [hunk 1]
```diff
@@ -48,23 +52,23 @@ title: "Školení PhpStormu - Tomáš Fejfar"
...
-        <strong>Recent files: </strong>Typicky mi ušetří přibližně 1 až 10 sekund oproti hledání v tabech - beru tedy 4s. Použito {21598|number:0:',':' '}&times;. Ušetřilo to tedy
-        <strong>{21598*4/60/60|number:0:',':' '} hodin</strong>
+        <strong>Recent files: </strong>Typicky mi ušetří přibližně 1 až 10 sekund oproti hledání v tabech - beru tedy 4s. Použito {{ 21598|number_format(0, ',', ' ') }}&times;. Ušetřilo to tedy
+        <strong>{{ (21598*4/60/60)|number_format(0, ',', ' ') }} hodin</strong>
     </li>
 </ul>
 <p>Celkově ušetřeno:</p>
 <div class="alert alert-danger">
-   <h5>61 hodin, tedy {61/8|number:1:',':' '} pracovních dní!</h5>
+    <h5>61 hodin, tedy {{ (61/8)|number_format(1, ',',' ') }} pracovních dní!</h5>
```
Comment (TomasVotruba) [inline source/skoleni/phpstorm.twig:73]: TIL: non-whole numbers have to be in brackets `(61/8)`

**Comment by TomasVotruba** [comment]
Is ready. Could you review?

**Comment by TomasVotruba** [comment]
This is ready to merge

**Comment by tomasfejfar** [comment]
I was AFK for most of the week, I hope to get to this in the weekend....

**Comment by tomasfejfar** [comment]
Thx :)

**Review by TomasVotruba — COMMENTED** [review]
You're welcome :+1:

**Pull request merged**